avc_codec_registration.src.html

```diff
@@ -154,16 +160,53 @@
   <dt><dfn enum-value for=AvcBitstreamFormat>avc</dfn></dt>
   <dd>
     SPS and PPS data are not included in the bitstream and are instead emitted
-    via the {{VideoEncoder}} [[output callback]] as the
+    via the [=VideoEncoder/[[output callback]]=] as the
     {{VideoDecoderConfig.description}} of the
-    {{VideoDecoderConfig}}.
-    <var ignore=''>output_config</var>.
+    {{EncodedVideoChunkMetadata.decoderConfig}}.
 
     NOTE: This format is described in greater detail by [[iso14496-15]],
         section 5.3.3.1. This format is commonly used in .MP4 files, where the
         player generally has random access to the media data.
 </dl>
 
+EncodedVideoChunk data {#encodedvideochunk-data}
+================================================
+
+{{EncodedVideoChunk}} [=EncodedVideoChunk/[[internal data]]=] is expected to be
+an access unit as defined in [[ITU-T-REC-H.264]] section 7.4.1.2.
+
+NOTE: An access unit contains exactly one primary coded picture.
+
+If the bitstream is in {{AvcBitstreamFormat/avc}} format,
+[=EncodedVideoChunk/[[internal data]]=] is assumed to be in canonical format, as
+defined in [[iso14496-15]] section 5.3.2.
```

Comment:
I'm not totally happy with this part, "in avc/annexb format" is basically a synonym for the format of `[[internal data]]`, but this is how the two are being linked together. I'm open to suggestions that would clear up this duplication.

Reply:
I've failed to come up with something more concise. You could say "If the bitstream is configured to be avc format, ..." - maybe slightly improves it? I'm fine with what you have too.